tdrs-backend/tdpservice/settings/common.py

```diff
@@ -329,3 +330,26 @@ class Common(Configuration):
 
     # The number of seconds to wait for socket response from clamav-rest
     AV_SCAN_TIMEOUT = os.getenv('AV_SCAN_TIMEOUT', 30)
+
+# add_header Content-Security-Policy
+# "default-src 'none';
+# script-src 'self';
+# style-src 'self' 'unsafe-inline';
+# img-src 'self' data:;
+# font-src 'self';
+# connect-src 'self' *.cloud.gov;
+# manifest-src 'self';
+# object-src 'none';
+# frame-ancestors 'none';
+# form-action 'none';";
+    CSP_DEFAULT_SRC = ("'none'")
+    CSP_SCRIPT_SRC = ("'self'")
+    CSP_STYLE_SRC = ("'self'", "'unsafe-inline'")
```

Comment:
This will probably get flagged in OWASP as a new warning, like in #907. I wonder if it's because of inline styles in Django Admin?

Reply:
Ended up not needing a style source at all